### DISCUSSION TITLE
Hopefully support for nested object, it doesnt generate an example

### DISCUSSION THREAD

**Discussion by masterbater** [scalar/scalar#1642] [Ideas]
For elements in scribe
<img width="339" alt="image" src="https://github.com/scalar/scalar/assets/26339988/62a53b7a-8dd5-4e86-a776-75e3074820f9">
<img width="440" alt="image" src="https://github.com/scalar/scalar/assets/26339988/7c8b59f9-835d-45f7-a719-dbc386e56124">

For scalar in scribe
<img width="262" alt="image" src="https://github.com/scalar/scalar/assets/26339988/d45fc2c8-0166-4aeb-bf91-26ac6da36f56">
If I make it required it shows blank []
also array of object will show an example like this
<img width="197" alt="image" src="https://github.com/scalar/scalar/assets/26339988/55ee79b6-e1c4-4096-a564-cb2ad3eb6415">

**Comment by hanspagel** [comment]
Ha, that’s my kind of issue. :) Any chance you can share the OpenAPI specification? I’ll look into this.

**Reply by masterbater** [reply]
[basta.json](https://github.com/scalar/scalar/files/15308489/basta.json)
Here you go please change the file extension to yaml

**Comment by masterbater** [comment]
I think it did properly generate the properties of array but scalar could have used the default example instead of generating per field

properties:
readings:
type: array
description: ''
example:
- []
items:
type: object
properties:
serial:
type: string
description: ''
example: eos
manual:
type: boolean
description: ''
example: true
timezone:
type: string
description: ''
example: America/Costa_Rica
source:
type: string
description: ''
example: ipsa
value:
type: array
description: ''
example:
- 12942970.0
items:
type: number
device:
type: string
description: ''
example: modi
readingType:
type: string
description: ''
example: vero
date:
type: string
description: 'Must be a valid date in the format Y-m-d\TH:i:sP.'
example: '2024-05-14T06:11:08+00:00'
required:
- serial
- manual
- timezone
- source
- value
- device
- readingType
- date

**Reply by hanspagel** [reply]
I’ve created an issue here: https://github.com/scalar/scalar/issues/2008
